critiquebrainz/frontend/templates/event/entity.html

```diff
@@ -15,7 +15,7 @@ <h2 class="pull-left">
     </h2>
 
     {% if not my_review %}
-      <a id="write-review" href="{{ url_for('review.create', place=id) }}"
+      <a id="write-review" href="{{ url_for('review.create', entity_type='event', entity_id=id) }}"
```

Comment:
whoops, good catch here. It seems that this has happened in a number of places (e.g. #297). Would you be able to have another detailed look and ensure that it isn't incorrect in any other place?